background.ts

```diff
@@ -484,6 +484,13 @@ Electron.app.on('before-quit', async(event) => {
 
   try {
     await k8smanager?.stop();
+    try {
```

Comment:
In the spirit of trying to avoid relying on exceptions for control-logic, shouldn't we be wrapping these lines in a win32 check? And then maybe you could get rid of the catch handler, because on win32 there should be a handler.